src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests2.cs

```diff
@@ -5617,10 +5678,11 @@ public class C3 { }
         var e = comp.GetMember<NamedTypeSymbol>("E");
         var extension = (SourceNamedTypeSymbol)e.GetTypeMembers().Single();
         Assert.Equal("extension(N1.N2.C1/C2/C3)", extension.ComputeExtensionGroupingRawName());
+        Assert.Equal("extension(N1.N2.C1.C2.C3)", extension.ComputeExtensionMarkerRawName());
```

Comment:
Do we have tests for classes with the `file` modifier?

Reply:
We don't, but none of the changes in this PR relate to accessibility, so it doesn't feel too useful

Reply:
I wasn't thinking about accessibility but the fact that `file` classes have mangled names and I'm wondering if they are mangled in the content-based names as well?